### DISCUSSION TITLE
Add a button that allows download of static PDF files

### DISCUSSION THREAD

**Discussion by syeduali93** [dotnet/docfx#8211] [Q&A] [answered]
I'd like to place a button on the header/footer of the docfx generated site that iwhen clicked it downloads an already generated/saved PDF in the asset folder of the site. 

Tried to scour the web for documentation on how to go about this but I don't think this is documented that well.

Any feedback will be appreciated.

**Comment by paulushub** [comment, marked as answer]
For the footer, you can either 
use the `_appFooter` attribute in the  `globalMetadata` to change the footer contents 
**--or--**
use a customized template by modifying either `footer.tmpl.partial` or `_footer.liquid`.

The `_appFooter` example: Assuming the following

* the PDF file **testpdf.pdf**
* the PDF file is place in a sub directory **images** (which is part of the **resource** definition already created by DocFX)

```
"resource": [
  {
	"files": [
	  "images/**"
	]
  }
],

. . .

"globalMetadata": {
  "_appFooter": "<span><a href=\"./images/testpdf.pdf\" target=\"blank\">Download PDF</a><br/>Copyright &#169; 2015-2022 Test Company<br>Generated by <strong>DocFX</strong></span>"
},
```
The result is shown below:
![pdf](https://user-images.githubusercontent.com/809351/203520362-74563721-77d3-4b6c-afc9-09a87e70c985.png)
